lib/taskana-core/src/main/java/pro/taskana/spi/task/api/ReviewRequiredProvider.java

```diff
@@ -0,0 +1,41 @@
+package pro.taskana.spi.task.api;
+
+import pro.taskana.common.api.TaskanaEngine;
+import pro.taskana.task.api.models.Task;
+
+/**
+ * The ReviewRequiredProvider allows to implement customized behaviour which requests a review
+ * instead of completing a {@linkplain Task}.
+ */
+public interface ReviewRequiredProvider {
+
+  /**
+   * Provide the active {@linkplain TaskanaEngine} which is initialized for this TASKANA
+   * installation.
+   *
+   * <p>This method is called during TASKANA startup and allows the service provider to store the
+   * active {@linkplain TaskanaEngine} for later usage.
+   *
+   * @param taskanaEngine the active {@linkplain TaskanaEngine} which is initialized for this
+   *     installation.
```

Comment:
Please remove the point at the end according to our guidelines

Reply:
done